ios/ios_project_setup.md

```diff
@@ -1,9 +1,11 @@
 # How To Create A New Project
 ## Create Xcode project and Github repo
-1. Create repo on Github with `.gitignore` and `README.md`. Usually, repos should be named with the format `<project-name>-ios` e.g `bose-stetson-ios`
+1. Create repo on Github with `.gitignore` and `README.md`. Repositories should be named with the format `<project-name>-ios` e.g `bose-stetson-ios`. The `-ios` suffix is important for other services.
 
 1. Create Xcode Project
 
+1. Ensure your Xcode Project is in the top level of the git repo directory
```

Comment:
Keolis was inherited with a poor project structure that has caused us some pains in the past. This doc is supposed to explain how to set up a new Xcode project and repository, so I feel like the instructions should point to what is easiest for us.